src/TestProblem/ELBDM/GaussianWavePacket/Init_TestProb_ELBDM_GaussianWavePacket.cpp

```diff
@@ -188,9 +223,51 @@ void SetGridIC( real fluid[], const double x, const double y, const double z, co
       break;
    }
 
+   const double Re = AnalyticalSolution_GaussianWavePacket( r, Time, 1 );
+   const double Im = AnalyticalSolution_GaussianWavePacket( r, Time, 2 );
```

Comment:
Define some constants like const int GAUSSIAN_TARGET_VAR_REAL = 1; const GAUSSIAN_TARGET_VAR_IMAG = 2; const int GAUSSIAN_TARGET_VAR_PHAS = 3; in this file maybe for clarity.

Reply:
Done.